redacted-compiler-gradle-plugin/src/main/kotlin/io/sweers/redacted/gradle/RedactedGradleSubplugin.kt

```diff
@@ -19,9 +19,10 @@ class RedactedGradleSubplugin : KotlinGradleSubplugin<AbstractCompile> {
 
   override fun getPluginArtifact(): SubpluginArtifact =
       SubpluginArtifact(
-          groupId = "io.sweers.redacted",
-          artifactId = "redacted-compiler-plugin",
-          version = "0.0.1" // TODO: What's the best way to keep this synced?
+          groupId = "dev.zacsweers.redacted",
+          artifactId = "redacted-compiler-plugin-gradle",
```

Comment:
I suspect it's silently doing nothing because it's not actually pulling the subplugin artifact.

```suggestion
          artifactId = "redacted-compiler-plugin",
```

Reply:
Ahh, I didn't realize this was in reference to the compiler plugin and not the gradle plugin. Made some progress now, though interestingly it appears that the kotlin plugin will try to add this as a full dependency (and thus try to fetch remotely). Going to see if I can get it to work by replacing its lookup in dependencySubstitution

Reply:
Substitution worked!